src/index.html

```diff
@@ -7,7 +7,53 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="container">
+    <div class="stars stars--0">
```

Comment:
These could be `ul` with `li`'s, don't use div's for everything, try to use semantic html